src/desktop/main.cpp

```diff
@@ -226,14 +226,6 @@ void DrawpileApp::deleteAllMainWindowsExcept(MainWindow *win)
 	}
 }
 
-QString DrawpileApp::greeting()
```

Comment:
I didn’t like this forced private information disclosure so I am glad it is gone now :-)

Reply:
Technical stuff like that isn't private information, at least not in the data protection sense. But I guess it does overdisclose the operating system, since it really doesn't matter for troubleshooting what exact OS you're running, the general platform would be enough.